vllm/worker/hpu_model_runner.py

```diff
@@ -649,12 +650,30 @@ def load_model(self) -> None:
                 assert hasattr(
                     self.model, "embedding_padding_modules"
                 ), "Model does not have embedding_padding_modules"
+
+                if supports_multimodal(self.model):
```

Comment:
Is this code coming from rebased version of "model_runner.py"? or is this something we have added on our own?